templates/deno/remix.config.js

```diff
@@ -1,12 +1,13 @@
 module.exports = {
   serverBuildTarget: "deno",
   server: "./server.ts",
-
-  /*
-  If live reload causes page to re-render without changes (live reload is too fast),
-  increase the dev server broadcast delay.
-
-  If live reload seems slow, try to decrease the dev server broadcast delay.
-  */
+  // If live reload causes page to re-render without changes (live reload is too fast),
+  // increase the dev server broadcast delay.
+  // If live reload seems slow, try to decrease the dev server broadcast delay.
```

Comment:
I prefer a multi-line comment here. Could we keep as-is?

Reply:
We have these in all other templates as well

Reply:
I don't see the value in making comment style consistent. If I had to pick one, I would argue that multi-line comments are better for comments spanning multiple lines (e.g. they can be edited without needing to add or remove intermediate `//`).

Can you keep this as-is?

Reply:
Don't have a strong opinion, just wanted to make it consistent.